### PR TITLE
Added an in-game client tooltip with IP and country

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/3564.

image

This does not add latency on purpose. See the discussion at https://github.com/OpenRA/OpenRA/pull/6752.

### CODE REVIEW

**Comment by RoosterDragon** [comment]
:+1:

File: mods/cnc/chrome/tooltips.yaml [hunk 1]
```diff
@@ -204,6 +204,30 @@ Background@CLIENT_TOOLTIP:
 			Height: 10
 			Font: TinyBold
 
+Background@INGAME_CLIENT_TOOLTIP:
+	Logic: ClientTooltipLogic
+	Background: panel-black
+	Height: 37
```
Comment (obrakmann) [inline mods/cnc/chrome/tooltips.yaml]: Please make this 35, 37 looks wrong:

35 screenshot from 2015-12-10 22 19 59 vs. 37 screenshot from 2015-12-10 22 20 35

**Comment by obrakmann** [comment]
Other than that, :+1:

**Comment by Mailaender** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/c046638658dd53aa0b6a8a722a7e452a43ab0bfa)

**Pull request merged**